mobile/src/main/res/values/strings.xml

```diff
@@ -416,4 +416,10 @@
     <string name="item_update_widget_auto_generate_widget_label">Auto generate widget label</string>
     <string name="item_update_widget_widget_label">Widget label</string>
     <string name="item_update_widget_default_label"><![CDATA[<Item label>: <Action>]]></string>
+    <string name="data_saver_title">Data saver</string>
+    <string name="data_saver_on">Reduce data usage on mobile data</string>
+    <string name="data_saver_off">Respect device setting</string>
+    <string name="data_saver_webview">WebView Items are not rendered if data saver is active</string>
+    <string name="data_saver_load">Load anyway</string>
+    <string name="data_saver_hint">%s is not loaded to save data</string>
```

Comment:
Should have a meaningful fallback for when the widget has no label ... and probably is -> was.

Reply:
`widget.label` always returns non-null. For Image items without label is returns something like "Image Url [-]".
`%s` can be replaced by the widget type if no label is present: "Image is not loaded to save data".

Reply:
The server always supplies a label for charts, even if the user hasn't set one. I'll check if the server does so for images, videos and mapviews as well.

Reply:
The widget type is now shown instead of the label.